web/yo/app/scripts/app.js

```diff
@@ -128,28 +128,34 @@ angular.module('oncokbApp').run(
             $rootScope.internal = true;
             $rootScope.meta = {
                 levelsDesc: {
-                    '0': 'FDA-approved drug in this indication irrespective of gene/variant biomarker',
+                    '0': '',
```

Comment:
These levels exist but should not get a description since we never define them in the paper.